tf_quant_finance/black_scholes/approximations/american_option.py

```diff
@@ -350,7 +350,7 @@ def bjerksund_stensland(*,
                         dividend_rates: types.RealTensor = None,
                         discount_factors: types.RealTensor = None,
                         is_call_options: types.BoolTensor = None,
-                        modified_boundary: bool = False,
+                        modified_boundary: bool = True,
                         dtype: tf.DType = None,
                         name=None) -> types.RealTensor:
   """Computes prices of a batch of American options using Bjerksund-Stensland.
```

Comment:
please update Raises section of the function. Also, add 

name: str = None

Reply:
Updated.  I also updated the example code, as it was no longer relevant.